README.md

```diff
@@ -86,8 +86,7 @@ Install dependencies [(See below)](#Dependencies)
 ```sh
 sudo rm -rf "/usr/local/share/ani-cli" "/usr/local/bin/ani-cli" "/usr/local/bin/UI" /usr/local/bin/player_* #If some of these aren't found, it's not a problem
 git clone "https://github.com/pystardust/ani-cli.git" && cd ./ani-cli
-sudo cp ./bin/ani-cli /usr/local/bin
-sudo cp -a ./lib/ani-cli /usr/local/lib
+sudo cp ./ani-cli /usr/local/bin
```

Comment:
tiny question... `/usr/local/*`? Shouldn't it be` /usr/` so that it could 
1. Better follow the LSB (most distros ive used that do updates that way do NOT use it for local installs)
2. Work with the Arch File Hierarchy
Either way people that need to do local installs through `/usr/local` know that and will modify the scripts to their liking.

Reply:
> Ah, sorry, I didnt see that it was already in the repository from before, ill send a MR after this gets merged since its kinda unrelated to this MR.

Microsoft calls it PR

Reply:
Im not microsoft :wink:

Reply:
this looks good